WinCompatibilityPack/WinCompatibilityPack.psd1

```diff
@@ -49,7 +49,28 @@ PowerShellVersion = '6.0'
 # RequiredModules = @()
 
 # Assemblies that must be loaded prior to importing this module
-# RequiredAssemblies = @()
+RequiredAssemblies = @(
+    'Microsoft.Win32.SystemEvents.dll'
+    'System.CodeDom.dll'
+    'System.Configuration.ConfigurationManager.dll'
+    'System.Data.DataSetExtensions.dll'
+    'System.Data.Odbc.dll'
+    'System.Diagnostics.EventLog.dll'
+    'System.Diagnostics.PerformanceCounter.dll'
+    'System.DirectoryServices.AccountManagement.dll'
+    'System.DirectoryServices.dll'
+    'System.DirectoryServices.Protocols.dll'
+    'System.Drawing.Common.dll'
+    'System.IO.Pipes.AccessControl.dll'
+    'System.IO.Ports.dll'
+    'System.Management.dll'
+    'System.Memory.dll'
+    'System.Runtime.Caching.dll'
+    'System.Runtime.CompilerServices.Unsafe.dll'
+    'System.Security.Cryptography.ProtectedData.dll'
+    'System.Security.Cryptography.Xml.dll'
+    'System.ServiceModel.Syndication.dll'
+)
```

Comment:
Is this list complete?  I believe WCP RC1 added some new assemblies

Reply:
This is fresh and I just generated it this morning. I took 6.0.2 assemblies and WCS-rc1 assemblies. and WCP assemblies not already loaded by 6.0.2 I added here. So it is not all of the assemblies in WCP, but since we already load quite a few of these in PowerShell proper this is a subset.

I had plans to automate that in PSCoreWindowsComapt.. or I debated handling it programtically from the .psm1. I forget the exact reason why, but just listing all of them in `RequiredAssemblies` sometimes result in errors.

Hmm.. thinking on it, though, we load more of these in PowerShell Core on Windows than on !Windows, some of them might actually work on Linux. I used PSCore on windows to generate the list of already loaded assemblies. Withouth an OS Logic gate for psd1, maybe it is better to load them from the psm1...

Reply:
Since this is an early preview, perhaps this is sufficient if you believe these are the ones not already loaded by PSCore6 as we have time to make changes to this module